src/main/java/bob/task/Events.java

```diff
@@ -0,0 +1,63 @@
+package bob.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+import bob.exception.InvalidInputException;
+
+/**
+ * Tasks that start at a specific date/time and ends at a specific date/time.
+ */
+public class Events extends Task {
```

Comment:
```suggestion
public class Event extends Task {
```
Since the class represents a single entity, I find Event more intuitive, what do you think?